custom/enikshay/ucr/reports/tb_notification_register_private.json

```diff
@@ -0,0 +1,712 @@
+{
+    "domains": [
+        "enikshay",
+        "enikshay-reports-qa"
+    ],
+    "server_environment": [
+        "enikshay"
+    ],
+    "data_source_table": "episode_2b",
+    "report_id": "tb_notification_register_private",
+    "config": {
+        "description": "",
+        "title": "TB Notification Register Private (Patient Case List) (2B)",
+        "sort_expression": [],
+        "visible": true,
+        "configured_charts": [],
+        "aggregation_columns": ["doc_id"],
+        "filters": [
+            {
+                "compare_as_string": false,
+                "datatype": "datetime",
+                "required": false,
+                "slug": "date",
+                "field": "date_of_diagnosis",
+                "type": "date",
+                "display": "Date of Diagnosis"
+            },
+            {
+                "compare_as_string": false,
+                "datatype": "string",
+                "fields": [
+                    "person_owner_id",
+                    "diagnosing_facility_id"
+                ],
+                "slug": "village",
+                "field": "person_owner_id",
+                "type": "multi_field_dynamic_choice_list",
+                "choice_provider": {
+                    "include_descendants": true,
+                    "type": "location",
+                    "order_by_hierarchy": true,
+                    "show_full_path": true
+                },
+                "display": "Location"
+            },
+            {
+                "slug": "is_migrated",
+                "required": false,
+                "datatype": "integer",
+                "show_all": true,
+                "choices": [
+                    {
+                        "display": "Show only migrated from Nikshay",
+                        "value": 1
+                    },
+                    {
+                        "display": "Show only eNikshay",
+                        "value": 0
+                    }
+                ],
+                "field": "case_created_by_migration",
+                "type": "choice_list",
+                "display": "Filter migrated data"
+            },
+            {
+                "pre_value": [
+                    "confirmed_tb",
+                    "confirmed_drtb"
+                ],
+                "datatype": "string",
+                "display": null,
+                "field": "episode_type",
+                "type": "pre",
+                "slug": "episode_type"
+            },
+            {
+                "pre_value": 1,
```

Comment:
This is the only difference from the public notification report except less domains to built for and only for enikshay environment.